auraloss/freq.py

```diff
@@ -132,6 +134,7 @@ def __init__(
         self.reduction = reduction
         self.mag_distance = mag_distance
         self.device = device
+        self.prefilter = prefilter
```

Comment:
As I suggest below, let's change this to something like `self.user_prefilter` or another name, maybe `self.custom_prefilter`, which is totally separate from `self.prefilter`, which will be used for the default perceptual weighting.